src/UI/citizens/components/submit-request/CitizensSubmitRequestStyle.js

```diff
@@ -0,0 +1,169 @@
+import styled from 'styled-components';
+
+const Container = styled.div`
+  position: relative;
+  width: 1024px;
+  height: 263px;
+  margin-left: 208px;
+  margin-top: 32px;
+  border-radius: 4px;
+  border: solid 1px #d9d9d9;
+  background-color: #ffffff;
+`;
+
+const Krug = styled.div`
+  margin: 16px;
+  width: 152px;
+  height: 152px;
+  opacity: 0.2;
+  border: solid 16px rgba(130, 212, 167, 0.4);
+  border-radius: 50%;
+`;
+
+const Broj = styled.div`
+  position: absolute;
+  top: 32px;
+  left: 16px;
+  width: 76px;
+  height: 145px;
+  opacity: 0.4;
+  font-family: Poppins-Regular;
+  font-size: 96px;
+  font-weight: 800;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #82d4a7;
+  margin-left: 38px;
+  text-align: center;
+`;
+
+const Title = styled.h1`
+  position: absolute;
```

Comment:
Umjesto da pozicioniraš h na ovaj način, trebalo je wrappati sav sadržaj te komponente sa nekim div-om i onda pozicionirati taj div na željeno mjesto. Unutar div-a će se nalaziti Title, Opis, Nesto itd. Ali onda nećeš morati pozicionirati svaki komponentu zasebno....